frontend/src/component/project/Project/CreateProject/CreateProject.tsx

```diff
@@ -111,18 +103,10 @@ const CreateProject = () => {
                 projectId={projectId}
                 setProjectId={setProjectId}
                 projectName={projectName}
-                projectMode={projectMode}
```

Comment:
For creation, we still want project mode. Is this removing project mode from creation?

Reply:
Furthermore, it seems to lack defaul value, which should be open

![image](https://github.com/Unleash/unleash/assets/964450/f3057b50-98d4-48a3-afda-847f16c614ce)

Reply:
Also, the dropdown seems to be unselectable.